packages/client/utils/AtlassianManager.ts

```diff
@@ -383,7 +387,9 @@ export default abstract class AtlassianManager {
     }
     if ('errors' in error) {
       const errorFieldName = Object.keys(error.errors)[0]
-      return new Error(`${errorFieldName}: ${error.errors[errorFieldName]}`)
+      if (errorFieldName) {
+        return new Error(`${errorFieldName}: ${error.errors[errorFieldName]}`)
```

Comment:
since this only conditionally returns now, the final return is of type T | JiraFieldError, which is why L605 had to be changed.

easy fix: 
```
const errorFieldName = Object.keys(error.errors)[0] ?? 'Unknown'
```